### ISSUE TITLE
The HUD overlays the taskbar

### ISSUE THREAD

**Issue by juangarciatn** [drweissbrot/cs-hud#202]
![image](https://github.com/user-attachments/assets/a0b11272-6617-4d70-b19f-12c83e8c2e14)

Doesnt show the taskbar

![image](https://github.com/user-attachments/assets/a8764ad8-e338-4471-b8da-0973a403e433)

**Comment by drweissbrot** [comment]
Yeah, it's supposed to do that :D The idea is that you'd run the game in fullscreen windowed, then have the overlay on top of it, also in fullscreen.

I think at least in Gnome (so probably in Cinnamon too) you can set up a keyboard shortcut to "restore" the window, which might un-fullscreen it, but don't know if that'd work for you.  
If you want to run CS in a window, it's probably easier if you go with an OBS browser source?
